frontend/talentsearch/resources/js/components/workPreferencesForm/WorkPreferencesForm.tsx

```diff
@@ -0,0 +1,156 @@
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-nocheck
+import React from "react";
+import { useIntl } from "react-intl";
+import { toast } from "react-toastify";
+import { errorMessages } from "@common/messages";
+import { BasicForm, Checklist, RadioGroup } from "@common/components/form";
+import { getLocale } from "@common/helpers/localize";
+import { notEmpty } from "@common/helpers/util";
+import {
+  OperationalRequirement,
+  useGetOperationalRequirementsQuery,
+} from "../../api/generated";
+import ProfileFormWrapper from "../applicantProfile/ProfileFormWrapper";
+import ProfileFormFooter from "../applicantProfile/ProfileFormFooter";
+
+export const WorkPreferencesForm: React.FunctionComponent<{
+  operationalRequirements: OperationalRequirement[];
+  handleSubmit: (data: any) => Promise<void>;
+}> = ({ operationalRequirements, handleSubmit }) => {
+  const intl = useIntl();
+  const locale = getLocale(intl);
+
+  function bold(msg: string) {
+    return <span data-h2-font-weight="b(700)">{msg}</span>;
+  }
+  function u(msg: string) {
+    return (
+      <span
+        // data-h2-border="b(black, bottom, solid, s)"
+        style={{
+          borderBottom: "1px solid #333",
+        }}
+      >
+        {msg}
+      </span>
+    );
+  }
+  const preferencesItems: { value: string; label: string }[] =
+    operationalRequirements.map((requirement) => {
+      return {
+        value: requirement.id,
+        label:
+          requirement.name[locale] ||
+          intl.formatMessage({
+            defaultMessage: "Error: operational requirement name not found.",
+            description:
+              "Error message if OperationalRequirement name is not defined.",
+          }),
+      };
+    });
+  return (
+    <ProfileFormWrapper
+      description="Certain jobs require you to work odd hours or perform tasks that are a little outside of the normal. Please indicate which special requirements you are comfortable with."
+      title="Work preferences"
+      crumbs={[
```

Comment:
I think part of the scope is to create a routing helper function, right?  You can create some placeholders and wire them up here.

Reply:
I may need help for this. I will try myself first and let you know.

Reply:
Yeah, this is what Tristan meant by _Create a applicantProfileRoutes file, a sibling to talentSearchRoutes._  You can see that file here: https://github.com/GCTC-NTGC/gc-digital-talent/blob/main/frontend/talentsearch/resources/js/talentSearchRoutes.ts